doc/CMakeLists.txt

```diff
@@ -32,6 +33,14 @@ IF(DEAL_II_COMPONENT_DOCUMENTATION)
     PATTERN "CMakeLists.txt" EXCLUDE
     PATTERN "doxygen" EXCLUDE
     PATTERN "*.in" EXCLUDE
+    PATTERN "news/changes.h" EXCLUDE
+    PATTERN "news/changes/*" EXCLUDE
+    )
+
+  INSTALL(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/news/changes.h
+    DESTINATION ${DEAL_II_DOCHTML_RELDIR}/news
+    COMPONENT documentation
```

Comment:
The reasoning was to create the same files as we did before. Thinking about it, we probably don't need to copy any files in `./doc/news`. I agree that all the information is contained in the files generated by doxygen already.